db/db.js

```diff
@@ -664,6 +664,7 @@ async function createLocation(locationid, phonenumber, movementThreshold, stilln
         alertApiKey,
         isActive,
         firmwareStateMachine,
+        sirenParticleId,
```

Comment:
make sure this matches the order of parameters in the query string, sirenParticleId should go after clientId

Reply:
Siren particle Id is second last in what I think is the query string - the reason why is I think that order was put in place somewhere when Theresa and I did the rebase (can't remember where though).

Reply:
oh! I think this was just my mistake oops